tests/model_hub_tests/models_hub_common/constants.py

```diff
@@ -3,6 +3,13 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import os
+import tempfile
+
+if os.environ.get('TFHUB_CACHE_DIR') is not None:
+    tf_hub_cache_dir = os.environ['TFHUB_CACHE_DIR']
+else:
+    tf_hub_cache_dir = os.path.join(tempfile.gettempdir(), "tfhub_modules")
+    os.environ['TFHUB_CACHE_DIR'] = tf_hub_cache_dir
```

Comment:
Doublecheck: do you really need to update environment variable? I'm not sure I understand the reason if you use a tf_hub_cache_dir

Reply:
Just to secure that this env variable is set up for sure. Most likely, TF Hub will always set it up but I did not find that they guarantee it for sure.